skglm/estimators.py

```diff
@@ -914,7 +914,7 @@ def fit(self, X, y):
             self.coef_ = np.empty([n_classes, X.shape[1]])
             self.intercept_ = 0.
             multiclass = OneVsRestClassifier(self).fit(X, y)
-            self.coef_ = multiclass.coef_
+            self.coef_ = np.array([clf.coef_[0] for clf in multiclass.estimators_])
```

Comment:
To keep full compatibility, you can use the original code of scikit-learn:

```python
        coefs = [e.coef_ for e in self.estimators_]
        if sp.issparse(coefs[0]):
            return sp.vstack(coefs)
        return np.vstack(coefs)
```

I see that we were expecting some sparse matrix sometimes.